delphi/app/plugins/test_plugins.py

```diff
@@ -0,0 +1,36 @@
+"""
```

Comment:
Calling this test_plugins is confusing, rename to something like example_plugins